### PR TITLE
Add CrLf website to sites.js

### PR DESCRIPTION
**Pull request by croqaz**
Hi! I would like to join the Webring, please.
Let me know if everything is OK.

Not sure if I need to add the link to the webring before, or after I get accepted.
Cheers

### CODE REVIEW

**Comment by neauoire** [comment]
https://github.com/XXIIVV/Webring#join-the-webring

**Comment by croqaz** [comment]
Thank you and sorry for posting without reading all the instructions.
I added the webring icon in the header of the website.

My site is mostly a blog, but once I'll move the old notes and quotes and bookmarks, I'll start adding more wiki stuff.
I hope it might be some useful information for somebody.

**Comment by croqaz** [comment]
Sure! No problem :)

File: scripts/sites.js [hunk 1]
```diff
@@ -766,7 +766,7 @@ const sites = [
     url: 'https://crlf.site',
     langs: ['en'],
     rss: 'https://crlf.site/feed.xml',
-    title: 'Cr;Lf;',
+    title: 'CrLf',
```
Comment (ckipp01) [inline scripts/sites.js]: haha, sorry, I wasn't clear enough. The title you had should be fine, but I meant the object keys. The `url` is out of order alphabetically to the other keys :)

Comment (croqaz) [reply scripts/sites.js]: Yeah, sorry, it's totally different than what I thought I have to do :D 
Fixed now

**Pull request merged**